src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx

```diff
@@ -290,8 +290,11 @@ Note that not all metrics have their custom tags as dimensions. Currently, only
 
 ## Metadata collection [#metadata-collection]
 
-Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch. This is an optional capability that's complementary to the CloudWatch Metric Streams integration.
+Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch.
 
+  <Callout variant="important">
+This is an optional capability that's complementary to the CloudWatch Metric Streams integration. However, it is necessary to have a full New Relic experience and not enabling it will lead to incomplete service discovery and monitorization.
+  </Callout>
```

Comment:
this is an important change

Reply:
my question here is, how deeply should we explain this? Should we go into details or this generic warning is enough?

Reply:
<img width="1139" alt="Screenshot 2023-01-30 at 09 17 42" src="https://user-images.githubusercontent.com/113110223/215423673-3015479f-c9b9-488c-9b14-4f4e3d136357.png">

Reply:
^^ the screenshoot is outdated, please check the source code directly